### PR TITLE
uses the united healthcare data for copd and diabetes

### PR DESCRIPTION
**Pull request by joshzarrabi**
- its a stasitically correct processed version of the brfss data
- properly hides states without enough data

Resolves https://github.com/SatcherInstitute/health-equity-tracker/issues/697

### CODE REVIEW

**Review by kkatzen — COMMENTED** [review]
Are you changing the data source from BRFSS to United Healthcare data? If so, I would rename the files

**Comment by joshzarrabi** [comment]
> Are you changing the data source from BRFSS to United Healthcare data? If so, I would rename the files

Yeah thats a good point I wasn't sure what to do. It is still BRFSS data, but it was processed by UHC and I got it from their site.

**Review by vanshkumar — COMMENTED** [review]
Looks good!

Two questions:
- Did you check that the definition of "Other" for this dataset matches ACS? If not, it should be OTHER_NONSTANDARD and we shouldn't be joining with ACS pop for this one
- Have you double checked that all the numbers match the united health foundation website? Interestingly, for COPD, I don't see many disparities for typically marginalized groups:
Screen Shot 2021-04-30 at 8 36 52 AM

File: frontend/src/data/config/MetricConfig.ts [hunk 1]
```diff
@@ -3,14 +3,12 @@ export type MetricId =
...
-  | "diabetes_count"
   | "diabetes_per_100k"
   | "diabetes_pct_share"
-  | "diabetes_count_share_of_known"
-  | "copd_count"
   | "copd_per_100k"
   | "copd_pct_share"
-  | "copd_count_share_of_known"
+  | "copd_pct"
+  | "diabetes_pct"
```
[the remark this answers is not in the document]
Comment (joshzarrabi) [reply frontend/src/data/config/MetricConfig.ts:11]: Sure, I'll just sort them

File: frontend/src/data/variables/BrfssProvider.ts [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
       "diabetes_per_100k",
       "diabetes_pct_share",
-      "diabetes_count_share_of_known",
-      "copd_count",
       "copd_per_100k",
       "copd_pct_share",
-      "copd_count_share_of_known",
       "brfss_population_pct",
+      "copd_pct",
+      "diabetes_pct",
```
[the remark this answers is not in the document]
Comment (joshzarrabi) [reply frontend/src/data/variables/BrfssProvider.ts:27]: I'll also sort them

**Comment by joshzarrabi** [comment]
> Looks good!
> 
> Two questions:
> 
> * Did you check that the definition of "Other" for this dataset matches ACS? If not, it should be OTHER_NONSTANDARD and we shouldn't be joining with ACS pop for this one
> * Have you double checked that all the numbers match the united health foundation website? Interestingly, for COPD, I don't see many disparities for typically marginalized groups:
>   Screen Shot 2021-04-30 at 8 36 52 AM


@vanshkumar For `OTHER`, they are using the same `OTHER` as `BRFSS` which we saw was the same as ACS.

And I did check to make sure the numbers are correct but obviously a second pair of eyes never hurt anyone :).

**Review by vanshkumar — APPROVED** [review]

**Pull request merged**